.github/workflows/e2e-reusable.yml

```
@@ -0,0 +1,121 @@
+name: integration-test
```

```suggestion
name: End-to-end Tests
```
End-to-end tests are more specific than integration tests.